UM/Settings/ContainerStack.py

```diff
@@ -832,6 +871,11 @@ def __str__(self) -> str:
     def __repr__(self) -> str:
         return str(self)
 
+    def __del__(self) -> None:
+        # None of the 'parents' seem to have __del__, so OK not to call `super.del` here.
```

Comment:
Should we not still do it, in case a parent someday gets a `__del__` ? In the meantime it should not hurt (applies multiple times)

Reply:
I tried that, but then it complains that `super` doesn't have a method `__del__`.

See also https://stackoverflow.com/questions/36722390/python-3-super-del

(... but there is a bit of a workaround if we want that, so I'll just commit that.)

Reply:
Oh, that sucks... You can keep it as is then.